src/bridge/IrcBridge.ts

```diff
@@ -454,6 +452,8 @@ export class IrcBridge {
             this.ircServers.push(server);
         }
 
+        this.clientPool = new ClientPool(this, this.dataStore);
```

Comment:
I don't see why move this?

Reply:
it has a (somewhat hard to see) dependency on the configured server list (provided by `IrcBridge.getServers()`).

Reply:
Oh except I failed to mention that this PR has a dependency on #1091